protobuf-kafkaconnect-converter/src/test/java/com/amazonaws/services/schemaregistry/kafkaconnect/protobuf/ToProtobufTestDataGenerator.java

```diff
@@ -351,6 +355,47 @@ private static Map<String, Schema> getTimeTypes() {
                 .build();
     }
 
+    public static Schema getDecimalSchema(String name) {
+        return createConnectSchema(name, getDecimalTypes(), ImmutableMap.of());
+    }
+
+    @SneakyThrows
+    public static DynamicMessage getProtobufDecimalMessage() {
+        Descriptors.FileDescriptor fileDescriptor = getDecimalFileDescriptor();
+        Descriptors.Descriptor descriptor = fileDescriptor.getMessageTypes().get(0);
+        DynamicMessage.Builder dynamicMessageBuilder = DynamicMessage.newBuilder(descriptor);
+
+        Decimals.Decimal.Builder decimalBuilder = Decimals.Decimal.newBuilder();
+        decimalBuilder.setUnits(1234);
+        decimalBuilder.setFraction(567890000);
```

Comment:
Please add more test-cases like "1234.56789134", "1234.567899999" and try to use smaller `scale` to see how it is interpreted? Do we also need to add any validations?

Reply:
I will add more tests.

Reply:
Added decimal with a large scale, and a test case with no decimal at all (scale 0)